src/load_stats.py

```diff
@@ -16,36 +63,25 @@ def load_stats(output_file):
     #
 
     stats = {site: dict() for site in config.STATS_SITES}
+    pool = multiprocessing.Pool()
```

Comment:
The main loop adds a significant load, so it's probably more reasonable to spare 1 cpu outside the Pool.